db/migrate/20231108093052_create_posts.rb

```diff
@@ -7,7 +7,7 @@ def change
       t.integer :likes_counter, default: 0
 
       # Add the author_id column to reference the users table
-      t.references :author, null: false, foreign_key: { to_table: :users }
+      t.references :user, null: false, foreign_key: true
```

Comment:
- You should also remove this as it doesn't follow the erd structure for the database.

Reply:
please explain how

Reply:
@stino-x , [ERD DIAGRAM](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png). You have changed your code and now it doesn't follow the project structure. see the post model, it has the authorId which references the users' table. You have changed it to be userId which does follow that for the post model. Kindly update your code to meet with this erd diagram database structure for the post model.